cli/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ _Released 9/24/2024 (PENDING)_
 
 **Misc:**
 
+- Cypress now consumes [geckodriver](https://firefox-source-docs.mozilla.org/testing/geckodriver/index.html) to help automate the Firefox browser instead of [marionette-client](https://github.com/cypress-io/marionette-client). Addresses [#30217](https://github.com/cypress-io/cypress/issues/30217). Addressed in [#30250](https://github.com/cypress-io/cypress/pull/30250).
```

Comment:
I don't think you need both the Addresses and Addressed in. I think Addressed in is only there if there's no corresponding issue for the PR

Reply:
I think semantic pull request was complaining about it because I wanted the 'addressed in', but complained that I didnt link the issue. I should be able to remove the later

Reply:
removed the 'addressed in' in d7fba61c6f